app/pb_kits/playbook/pb_textarea/_textarea.html.erb

```diff
@@ -1,26 +1,17 @@
 <%= content_tag(:div,
-    id: object.id,
+    aria: object.aria,
+    class: object.classname,
     data: object.data,
-    class: object.classname) do %>
+    id: object.id) do %>
   <% if object.label.present? %>
-    <%= pb_rails("caption", props: {text: object.label, dark: object.dark}) %>
+    <%= pb_rails("caption", props: {text: object.label}) %>
```

Comment:
I believe dark needs to be passed to the children. Otherwise setting dark would not set dark on the caption